src/components/look-at.js

```diff
@@ -50,71 +52,56 @@ module.exports.Component = registerComponent('look-at', {
       return object3D.lookAt(self.vector.setFromMatrixPosition(target3D.matrixWorld));
     }
 
-    // Query for the element, grab its object3D, then register a behavior on the scene to
-    // track the target on every tick.
-    if (targetSelector) {
-      targetEl = document.querySelector(targetSelector);
-      if (!targetEl) {
-        warn('"' + targetSelector + '" does not point to a valid entity to look-at');
-        return;
-      }
-      if (!targetEl.hasLoaded) {
-        return targetEl.addEventListener('loaded', function () {
-          self.beginTracking(targetEl);
-        });
-      }
-      return self.beginTracking(targetEl);
+    // No longer looking at anything (i.e., look-at="").
+    if (!target ||
+        (target.constructor === Object && !Object.keys(target).length)) {
+      return self.remove();
     }
 
     // Look at a position.
-    if (position) {
-      object3D.lookAt(new THREE.Vector3(position.x, position.y, position.z));
-      return;
+    if (target.constructor === Object) {
```

Comment:
how come? it's the same result, without a hardcoded string. will optimize better too